app/assets/stylesheets/home.scss

```diff
@@ -13,6 +13,16 @@ $white: #ffffff;
   min-height: 100%;
 }
 
+.results {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+.search {
+  background-color: #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`